src/services/utilServices/SendDNSRecordEmailClient.ts

```diff
@@ -0,0 +1,121 @@
+import { groupBy } from "lodash"
+
+export interface DnsRecordsEmailProps {
+  requesterEmail: string
+  repoName: string
+  domainValidationSource: string
+  domainValidationTarget: string
+  primaryDomainSource: string
+  primaryDomainTarget: string
+  redirectionDomainSource?: string
+  redirectionDomainTarget?: string
+}
+
+export interface LaunchFailureEmailProps {
+  // The fields here are optional since a misconfiguration in our
+  // formSG can cause some or even all fields to be missing
+  requesterEmail?: string
+  repoName?: string
+  primaryDomain?: string
+  error: string
+}
+
+const tableStyle =
+  "border-collapse: collapse; width: 100%; border: 1px solid #ddd; text-align: center;"
+
+const thStyle = "padding: 8px; text-align: center; border: 1px solid #ddd;"
+
+const tdStyle = "padding: 8px; text-align: left; border: 1px solid #ddd;"
+
+const headerRowStyle =
+  "background-color: #f2f2f2; border: 1px solid #ddd; text-align: center;"
+
+const repoNameStyle =
+  "padding: 8px; text-align: left; font-weight: bold; border: 1px solid #ddd;"
+
+export function getDNSRecordsEmailBody(
+  submissionId: string,
+  dnsRecordsEmailProps: DnsRecordsEmailProps[]
+) {
+  let html = `<p style="font-size: 16px; font-weight: bold;">Isomer sites are in the process of launching. (Form submission id [${submissionId}])</p>
```

Comment:
Nit: should it be abstracted to a style const since it's being used at least twice in this file?

Reply:
this is fair, modified it here 9e43fe51